### ISSUE TITLE
[Accessibility] ExtendedPeoplePicker does not announce suggestions when popped up

### ISSUE THREAD

**Issue by thuy-n-lu** [microsoft/fluentui#15983]
### Environment Information

- **Package version(s)**: 7.148.2
- **Browser and OS versions**: Windows OS version: 2004 (Build 20211.1000)
Edge dev version:  87.0.637.0 (Official build) dev (64-bit)

### Describe the issue:

Edit field announced as <combo edit, has popup>and on typing, the Suggestions popup appears but no announcement made by Narrator.
### Please provide a reproduction of the issue in a codepen:
https://fabricweb.z5.web.core.windows.net/oufr/7.148.2/#/examples/extendedpeoplepicker

#### Actual behavior:

Edit field announced as <combo edit, has popup>and on typing, the Suggestions popup appears but no announcement made by Narrator.
#### Expected behavior:

Appearance of the suggestions should be announced by Narrator.

#### Documentation describing expected behavior
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7bb19c05bd-d82f-4d6d-a8e5-7081bf5c2dba%7d

**Comment by paulgildea** [comment]
@thuy-n-lu Thanks for filing this issue as well.

@smhigley FYI
